esmvalcore/preprocessor/_area.py

```diff
@@ -419,7 +419,7 @@ def _get_masks_from_geometries(geometries, lon, lat, method='contains',
     if ids:
         ids = [str(id_) for id_ in ids]
     for i, item in enumerate(geometries):
-        for id_prop in ('name', 'NAME', 'id', 'ID'):
+        for id_prop in ('name', 'NAME', 'Name', 'id', 'ID'):
```

Comment:
Looks good to me, but I would like to propose this as an alternative option:

```suggestion
        for id_prop.lower() in ('name', 'id'):
```